homepairs/homepairsUI/App.tsx

```diff
@@ -20,27 +17,9 @@ import store from './src/state/store';
 
 const checkSession = async () => {
     await LoadFonts();
-    
-    await AsyncStorage.getItem('profile').then(profile => {
-        const storedAccountProfile= JSON.parse(profile);
-        const {properties} = storedAccountProfile;
-        store.dispatch(parseAccount(storedAccountProfile));
-        store.dispatch(fetchProperties(properties));
-    }).catch(() => {});
-
-    /*
-    await AsyncStorage.getItem('preferredProviders').then(preferredProviders => {
-        const json = JSON.parse(preferredProviders);
-        const {providers} = json;
-        const parsedProviders = parsePreferredProviders(providers);
-        store.dispatch(refreshServiceProviders(parsedProviders as ServiceProvider[]));
-    }).catch(() => {});
-    
-    await AsyncStorage.getItem('selectedProperty').then(selectedPropertyId => {
-        const storedSelectedPropertyId = selectedPropertyId;
-        store.dispatch(setSelectedProperty(storedSelectedPropertyId));
-    }).catch(() => {});
-    */
+    await AsyncStorage.getItem('persist:root').then(response => {
+        console.log(JSON.parse(response));
```

Comment:
let's clean these up next sprint

Reply:
Sounds like a plan. We can definitely have the main focus of the next sprint for clearing up extraneous comments, console logs, and for documentation.